### DISCUSSION TITLE
Keybinding for switching between the editor and the terminal.

### DISCUSSION THREAD

**Discussion by ubaldot** [VSCodeVim/Vim#7628] [Q&A]
When I was using `gvim `I use to have a vertical split with the editor in one split and a `vimterminal `on the other split running `IPython`.
I used to jump to/from the editor/terminal with `<c-w><c-w>`, now I have no clue how to that. 
Can someone help, please?

**Comment by sebslight** [comment]
Not Vim command but on VSCode have you tried `` CTRL + ` ``?

**Comment by ttytm** [comment]
hey @ubaldot,

the answer above toggles the terminal. 

To achieve the exact behavior you are looking for, you could can map it directly in your `keybindings.json`. For example, I have amped up the regular mappings to move to the next `h` | `j` | `k` | `l` window (e.g.   `<c-w>j` `<c-w>k` etc.) to just `<c-j>` `<c-k>` etc.. For the terminal it is necessary to specify an additional condition. You can similarly map it to your preferred keys.
```json
	{
		"key": "ctrl+j",
		"command": "workbench.action.focusBelowGroup",
		"when": "!editorFocus && !inSearchEditor && !inputBoxFocus"
	},
	{
		"key": "ctrl+k",
		"command": "workbench.action.focusAboveGroup",
		"when": "!editorFocus"
	},
	{
		"key": "ctrl+k",
		"command": "workbench.action.focusLastEditorGroup",
		"when": "terminalFocus"
	},
```
